### PR TITLE
LPS-176665 Replace alert divs with clay:alert taglib in asset-publisher-web

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://liferay.atlassian.net/browse/LPS-176665)

## Test Scenarios

**Case 1**
- Go to Site Builder > Pages > go to a content page 
- Add an Related Assets widget > see the message
- Add an Asset Publisher widget > see the message
- Go to the Asset Publisher widget's configuration > asset selection > select Manual 
- Go back to the widget view > see message 
Screenshot 2023-07-03 at 17 02 10
Screenshot 2023-07-04 at 10 15 55

**Case 2**
- Go to Content and data > create a Web Content
- Go to Site Builder > Pages > go to a content page 
- Add an Asset Publisher widget 
- Go to the Asset Publisher widget's configuration > asset selection > select Manual 
- Go to asset entries section > select basic web content > save
- Go to Content and data > delete the web content and remove it from the recycle bin
- Go back to Asset Publisher widget's configuration > asset entries > see message 
Screenshot 2023-07-03 at 16 44 34

**Case 3**
**Tested in code, cannot be tested on UI**
Screenshot 2023-07-03 at 17 07 32

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:relevant

**Comment by DiegoHu97** [comment]
ci:test:relevant

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
@victorg1991 Updated! 😄

**Comment by DiegoHu97** [comment]
ci:test:relevant

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/view.jsp [hunk 1]
```diff
@@ -122,15 +122,25 @@ if (assetPublisherDisplayContext.isEnableTagBasedNavigation() && !assetPublisher
...
 
 						<div>
-							<aui:a href="javascript:void(0);" onClick="<%= portletDisplay.getURLConfigurationJS() %>"><liferay-ui:message key="select-a-collection-to-make-it-visible" /></aui:a>
+							<clay:button
+								cssClass="p-0"
+								displayType="link"
+								onClick="<%= portletDisplay.getURLConfigurationJS() %>"
+								small="<%= true %>"
+							>
+								<liferay-ui:message key="select-a-collection-to-make-it-visible" />
```
Comment (victorg1991) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/view.jsp:142]: Can you use the label property in the clay button? there is no need to use an additional taglib

Comment (victorg1991) [reply modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/view.jsp:142]: :)

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
@victorg1991 Updated! 😄

**Comment by victorg1991** [comment]
Divino! 👏

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:forward:force

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/12715

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**